docs/Validating.md

```diff
@@ -76,28 +76,23 @@ btcli s register --netuid 168 --wallet.name [wallet_name] --wallet.hotkey [walle
 
 ## Validating
 
-Before launching your validator, log into wandb by running the following command and pasting your API key when prompted. If you do not have an API key, contact the BitMind team via Discord for support.
-```bash
-wandb login
-```
-
-
-You can launch your validator via pm2 using the following command. To stop your validator, you can run `pm2 delete validator`.
+You can launch your validator with `run_neuron.py`.
```

Comment:
[Question] What are the thoughts here on this naming convention as opposeed to explicitly running a miner

Reply:
`run_neuron.py` does the same exact steps for miners and validators, so it'd be redundant to split it into two separate scripts that only differ by 2-3 strings